Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedTestRunner.swift

```diff
@@ -17,10 +17,6 @@ struct UnifiedTestRunner {
         self.serverVersion = try self.internalClient.serverVersion()
         self.topologyType = try self.internalClient.topologyType()
         self.serverParameters = try self.internalClient.serverParameters()
-
-        // The test runner SHOULD terminate any open transactions using the internal MongoClient before executing any
-        // tests.
-        try self.terminateOpenTransactions()
```

Comment:
this only seems useful if there was a test that ran beforehand which left open transactions. I think the only case that might happen is right after the transactions tests run (unless if there are others leaving txns open that I'm not aware of). so it doesn't seem worth running this every time